customize.dist/translations/messages.pl.js

```diff
@@ -0,0 +1,229 @@
+
+    out.poll_closeWizardButton = "Zamknij kreator";
+    out.poll_closeWizardButtonTitle = "Zamyka kreator";
+    out.poll_wizardComputeButton = "Ustawienia kalkulacji";
+    out.poll_wizardClearButton = "Wyczyść tabelę";
+    out.poll_wizardDescription = "Automatycznie stwórz część opcji poprzez wpisanie ilości dat i godzin";
+    out.poll_wizardAddDateButton = "+ Daty";
+    out.poll_wizardAddTimeButton = "+ Godziny";
+
+    out.poll_addUserButton = "+ Użytkownik";
+    out.poll_addUserButtonTitle = "Naciśnij, by dodać użytkownika";
+    out.poll_addOptionButton = "+ Opcja";
+    out.poll_addOptionButtonTitle = "Naciśnij, by dodać opcję";
+    out.poll_addOption = "Zaproponuj opcję";
+    out.poll_optionPlaceholder = "Opcja";
+    out.poll_addUser = "Podaj imię";
+    out.poll_userPlaceholder = "Twoje imię";
+    out.poll_removeOption = "Jesteś pewien, że chcesz usunąć tę opcję?";
+    out.poll_removeOptionTitle = "Usuń wiersz";
+    out.poll_removeUser = "Jesteś pewien, że chcesz usunąć tego użytkownika?";
+    out.poll_removeUserTitle = "Usuń kolumnę";
+    out.poll_editOption = "Jesteś pewien, że chcesz edytować tę opcję?";
+    out.poll_editOptionTitle = "Edytuj wiersz";
+    out.poll_editUser = "Jesteś pewien, że chcesz edytować tego uzytkownika?";
+    out.poll_editUserTitle = "Edytuj kolumnę";
+
+    out.poll_titleHint = "Tytuł";
+    out.poll_descriptionHint = "Opis";
+
+    // index.html
+
+    out.main_p1 = 'CryptPad to edytor czasu rzeczywistego stawiający na wspólpracę i <strong>prostotę użytkowania</strong>. Szyfrowanie w twojej przeglądarce, chroni twoje dane przed serwerem, chmurą i ABW. Tajny klucz jest przechowywany w adresie strony <a href="https://pl.wikipedia.org/wiki/Uniform_Resource_Identifier#Pe.C5.82ny_wz.C3.B3r_URI_hierarchicznych.5B2.5D">identyfikator fragmentu w URI</a> którego nigdy nie przekazujemy serwerowi, a dostępny jest dla javascript. Udostępniając URL(link), przekazujesz innym upoważnienie do uczestnictwa.';
+    out.main_p2 = 'Ten projekt wykorzystuje wizualny edytor <a href="http://ckeditor.com/">CKEditor</a> , <a href="https://codemirror.net/">CodeMirror</a>, oraz silnik czasu rzeczywistego <a href="https://github.com/xwiki-contrib/chainpad">ChainPad</a>.';
+    out.main_howitworks = 'Jak to działa';
+    out.main_howitworks_p1 = 'CryptPad wykorzystuje wariant algorytmu <a href="https://en.wikipedia.org/wiki/Operational_transformation">Transformacji operacyjnej</a> który jest wstanie odnaleźć rozdzielony konsensus wprowadzanych danych. Używa do tego <a href="https://bitcoin.org/bitcoin.pdf">Łańcuch blokowy Nakamoto</a>, twór zpopularyzowany przez <a href="https://en.wikipedia.org/wiki/Bitcoin">Bitcoin</a>. W ten sposób algorytm może pominąć potrzebę centralnego serwera do rozwiązywania Konfliktów Operacji Przekształcania poprzez Edycję. Bez potrzeby rozwiązywania konfliktów, serwer może pozostać w niewiedzy o zawartości która jest edytowana w dokumencie.';
```

Comment:
To discuss.
Yeah. I had some issues on "distributed consensus", but as we have an exact equivalent of that word, though less common, I'd stick with it.
Disagree.
It all comes from SQL databases theory.
Konflikty operacji przekształcania is a correct term and I've added edycji, just to ensure people that come with "zero-knowledge" will understand what we are talking about.